src/test/java/Safron/Hotline/Tests/ChangeLocationTestPositive.java

```diff
@@ -1,31 +1,29 @@
-package Safron.Tests;
+package Safron.Hotline.Tests;
+
 import BasePageAndTestInit.TestInit;
-import Safron.Pages.HotlineHeaderViewElements;
-import Safron.Pages.HotlineHeaderViewHelper;
-import Safron.Pages.HotlineLocationViewElements;
-import Safron.Pages.HotlineLocationViewHelper;
+import Safron.Hotline.Pages.HotlineHeaderViewElements;
+import Safron.Hotline.Pages.HotlineHeaderViewHelper;
+import Safron.Hotline.Pages.HotlineLocationViewElements;
+import Safron.Hotline.Pages.HotlineLocationViewHelper;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
 public class ChangeLocationTestPositive extends TestInit {
-    //enter the correct location
+
     @Test
-    public void test1() {
+    public void testWithCorrectLocation() {
         openUrl("https://hotline.ua/");
```

Comment:
move openUrl() method on the start of the test (line 21)

Reply:
fix